Don't error out on file timestamps older than what zip supports.
https://trac.macports.org/ticket/63926
--- flit_core/wheel.py.orig	2024-11-03 03:21:55
+++ flit_core/wheel.py	2025-02-12 23:07:39
@@ -8,6 +8,7 @@
 import os
 import os.path as osp
 import stat
+import sys
 import tempfile
 from pathlib import Path
 from types import SimpleNamespace
@@ -109,7 +110,10 @@
             rel_path = rel_path.replace(os.sep, '/')
 
         if self.source_time_stamp is None:
-            zinfo = zipfile.ZipInfo.from_file(full_path, rel_path)
+            if sys.version_info[:2] >= (3, 8):
+                zinfo = zipfile.ZipInfo.from_file(full_path, rel_path, strict_timestamps=False)
+            else:
+                zinfo = zipfile.ZipInfo.from_file(full_path, rel_path)
         else:
             # Set timestamps in zipfile for reproducible build
             zinfo = zipfile.ZipInfo(rel_path, self.source_time_stamp)
